api/controllers/v0.3/endpoints.js

```diff
@@ -323,6 +323,7 @@ async function checkAddress(req, res) {
  * @param {HTTP response} res
  */
 async function createPatron(req, res) {
+  const usernameHasBeenValidated = req.body.usernameHasBeenValidated || false;
```

Comment:
Isn't `usernameHasBeenValidated` just a boolean value?  If so, why do you need the `|| false`?  `const usernameHasBeenValidated = !!req.body.usernameHasBeenValidated` seems more readable.

Reply:
Yeah, that's a good update.